GitUI/CommandsDialogs/SettingsDialog/Pages/ColorsSettingsPage.cs

```diff
@@ -27,6 +28,8 @@ protected override void OnRuntimeLoad()
                 SettingsToPage();
             }
 
+            _themeEditorController = new FormThemeEditorController(new ThemeManager(new DefaultTheme()), new ThemePersistence());
```

Comment:
`FormThemeEditor` is not modal, user can close SettingsDialog and proceed with editing theme, therefore it's lifetime cannot be bound to `ColorSettingsPage`

Reply:
Fair enough, however the handling must be resilient to race conditions when a user closes multiple instances of the app.
Theme settings must be saved when the dialog is closed.

Reply:
> Theme settings must be saved when the dialog is closed.
it is

Reply:
obsolete, `FormThemeEditor` was removed